python/src/aica_api/client.py

```diff
@@ -211,6 +221,10 @@ def unload_hardware(self, interface_name: str) -> requests.Response:
         """
         endpoint = 'application/hardware/' + interface_name
         return requests.delete(self._endpoint(endpoint))
+    
+    def get_state(self):
```

Comment:
Sorry, I made a mistake in the issue. This function should be `get_application` and it should use the `application` endpoint. Could you then also add a function docstring as above?